jobs/socs/esports-discord-bot.hcl

```diff
@@ -0,0 +1,38 @@
+job "esports-discord-bot" {
+  datacenters = ["aperture"]
+  type = "service"
+
+  group "esports-bot" {
+    count = 1
+
+    task "esports-bot" {
+      driver = "docker"
+
+      config {
+        image = "ghcr.io/aydenjahola/discord-multipurpose-bot:main"
+        force_pull = true
```

Comment:
Don't think we need this anymore
```suggestion
```

Reply:
yeah I copy pasted the job from gomlbot ngl, will remove now!